src/workerd/api/trace.h

```diff
@@ -71,6 +71,11 @@ class TraceItem final: public jsg::Object {
   kj::ArrayPtr<jsg::Ref<TraceException>> getExceptions();
   kj::ArrayPtr<jsg::Ref<TraceDiagnosticChannelEvent>> getDiagnosticChannelEvents();
   kj::Maybe<kj::StringPtr> getScriptName();
+  // TODO(someday): we expose this as jsg::Optional for now, since that ensures that the property will
```

Comment:
@jasnell I've gone for this approach to "hide" the `scriptVersionId` property for now when it's not explicitly set. I did briefly consider making it a `*_PROTOTYPE_PROPERTY` instead to further "hide" it as they don't show up when calling `Object.keys()` but that seems like it would be weirdly inconsistent with the other properties here. LMK what you think.

Reply:
This is fine I think.